Dockerfile

```diff
@@ -0,0 +1,15 @@
+# pull official base image
+FROM python:2.7-alpine
+
+# set work directory
+WORKDIR /usr/src/app
+
+COPY . /usr/src/app/
```

Comment:
probs not super important since this is just a demo app, but it's a common practice to copy the requirements before copying the entire app, and running:
```
RUN pip install --upgrade pip && pip install -r requirements.txt
```
That way you don't have to install all the dependencies every time your source code changes and take advantage of how docker caches each layer.

Reply:
awesome!

Reply:
Ya i had this orginally but then when i had to build the postgres deps i got lazy